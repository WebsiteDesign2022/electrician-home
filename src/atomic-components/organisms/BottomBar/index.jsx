
import React from 'react';

import { Nav, Navbar } from 'react-bootstrap';

import AuthorMark  from '../../molecules/AuthorMark';

import SocialMediaButtonGroup from '../../molecules/SocialMediaButtonGroup';

import './index.scss';

// TODO: Extract text-content. (Commented Eugene, 16APR22)
const BottomBar = () => (
    <Navbar bg="dark" variant="dark" className="navbar align-items-center">
        <Nav className="nav col-md-8 mx-4 author-mark">
            <AuthorMark text="Designed by ... | Powered by ... " />
        </Nav>
        
        <div className="col-md-3 justify-content-end">
            <SocialMediaButtonGroup />
        </div>
    </Navbar>
);

export default BottomBar;
