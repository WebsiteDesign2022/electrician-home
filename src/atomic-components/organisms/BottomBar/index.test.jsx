import TestRenderer from 'react-test-renderer';

import BottomBar from '.';

describe("BottomBar", () => {
    test("renders BottomBar", () => {
        const bottomBar = TestRenderer.create(
            <BottomBar/>
        );

        expect(bottomBar.toJSON()).toMatchSnapshot();
    });
});
