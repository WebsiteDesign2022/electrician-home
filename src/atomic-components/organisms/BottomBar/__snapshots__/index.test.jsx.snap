// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomBar renders BottomBar 1`] = `
<nav
  className="navbar align-items-center navbar navbar-expand navbar-dark bg-dark"
>
  <div
    className="nav col-md-8 mx-4 author-mark navbar-nav"
    onKeyDown={[Function]}
  >
    <div
      className="text-white"
    >
      Designed by ... | Powered by ... 
    </div>
  </div>
  <div
    className="col-md-3 justify-content-end"
  >
    <ul
      className="nav"
    >
      <li
        className="nav-item"
      >
        <a
          className="nav-link active"
          href="#"
        >
          Meta
        </a>
      </li>
      <li
        className="nav-item"
      >
        <a
          className="nav-link active"
          href="#"
        >
          Twitter
        </a>
      </li>
      <li
        className="nav-item"
      >
        <a
          className="nav-link active"
          href="#"
        >
          RSS
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
